src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -134,20 +132,7 @@ const UsersFragment = ({
   });
 
   const handleUser = async (from, willFollow, item) => {
-    if (from === FROM_FOLLOWED_USERS_INITIAL || from === FROM_UNFOLLOWED_USERS_INITIAL) {
-      updateFollowDiscoveryContext(willFollow, item);
-    }
-
-    if (from === FROM_FOLLOWED_USERS) {
-      const newFollowedUsers = [...followedUsers];
-      exhangeFollower(newFollowedUsers, willFollow, item.user ? item.user.user_id : item.user_id);
-      setFollowedUsers(newFollowedUsers);
-    }
-    if (from === FROM_UNFOLLOWED_USERS) {
-      const newUnfollowedUsers = [...unfollowedUsers];
-      exhangeFollower(newUnfollowedUsers, willFollow, item.user ? item.user.user_id : item.user_id);
-      setUnfollowedUsers(newUnfollowedUsers);
-    }
+    updateFollowDiscoveryContext(willFollow, item);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The refactoring of `handleUser` function seems to simplify the logic by removing the conditionals and directly updating the follow discovery context. However, please ensure that this change doesn't affect the expected behavior of the application as the previous implementation had different actions based on the `from` parameter.



<!-- This is an auto-generated comment by OSS CodeRabbit -->